src/controllers/donorController.ts

```diff
@@ -48,19 +41,22 @@ export const postDonors = (req: Request, res: Response) => {
  * @route PUT /donors/:donor_id
  */
 export const modifyDonor = (req: Request, res: Response) => {
-    const id = req.params.donor;
-    const updatedDonor = req.params;
-    return Donor.findByIdAndUpdate(id, updatedDonor)
-        .then(result => res.status(200).json({ success: true }))
-        .catch((error: Error) => res.status(400).json({ success: false, message: error.message }));
+    const id = req.params.donor_id;
+    return User.findByIdAndUpdate(id, req.body)
+        .then(result => {
+            return res.status(200).json({
+                donor: result
+            });
+        })
+        .catch((error: Error) => res.status(400).json({ message: error.message }));
 };
 
 /**
  * Gets Donations
  * @route GET /donations
  */
 export const getDonations = (req: Request, res: Response) => {
-    Donation.find().populate('donor', '_id name latitude longitude')
+    Donation.find().populate('user', '_id donorInfo.name donorInfo.address')
```

Comment:
Can you make this populate the longitude and latitude, in addition to the address?